src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0993.xml

```diff
@@ -1,765 +1,755 @@
+                                </brick>
+                                <brick type="ThinkBubbleBrick">
+                                    <commentedOut>false</commentedOut>
+                                    <formulaList>
+                                        <formula category="STRING">
+                                            <type>STRING</type>
+                                            <value>Hmmmm!</value>
+                                        </formula>
+                                    </formulaList>
+                                    <type>1</type>
+                                </brick>
+                                <brick type="SayForBubbleBrick">
+                                    <commentedOut>false</commentedOut>
+                                    <formulaList>
+                                        <formula category="DURATION_IN_SECONDS">
+                                            <type>NUMBER</type>
+                                            <value>1.0</value>
+                                        </formula>
+                                        <formula category="STRING">
+                                            <type>STRING</type>
+                                            <value>Hello!</value>
+                                        </formula>
+                                    </formulaList>
+                                    <type>0</type>
+                                </brick>
+                                <brick type="ThinkForBubbleBrick">
+                                    <commentedOut>false</commentedOut>
+                                    <formulaList>
+                                        <formula category="DURATION_IN_SECONDS">
+                                            <type>NUMBER</type>
+                                            <value>1.0</value>
+                                        </formula>
+                                        <formula category="STRING">
+                                            <type>STRING</type>
+                                            <value>Hmmmm!</value>
+                                        </formula>
+                                    </formulaList>
+                                    <type>1</type>
+                                </brick>
+                                <brick type="ChooseCameraBrick">
+                                    <commentedOut>false</commentedOut>
+                                    <spinnerSelectionID>0</spinnerSelectionID>
+                                </brick>
+                                <brick type="ChooseCameraBrick">
+                                    <commentedOut>false</commentedOut>
+                                    <spinnerSelectionID>1</spinnerSelectionID>
+                                </brick>
+                                <brick type="HideTextBrick">
+                                    <commentedOut>false</commentedOut>
```

Comment:
As far as I can see from this changelog the `Y_POSITION` and `X_POSITION` values have been removed. Although  I agree that those tags do not have a lot of use, we still need them since Catroid uses them at the serialization.